src/envoy/http/authn/http_filter_integration_test.cc

```diff
@@ -104,7 +104,7 @@ TEST_P(AuthenticationFilterIntegrationTest, EmptyPolicy) {
 
 TEST_P(AuthenticationFilterIntegrationTest, SourceMTlsFail) {
   config_helper_.addFilter(R"(
-    name: istio_authn
+    name: istio.authn
     config:
```

Comment:
Is it possible to cover both old name and new name? Once we deprecate the old name, we remove the test as well.